prunner.go

```diff
@@ -362,14 +363,15 @@ func (r *PipelineRunner) HandleTaskChange(t *task.Task) {
 		end := t.End
 		jt.End = &end
 	}
-	jt.Errored = t.Errored
-	jt.Error = t.Error
 	jt.ExitCode = t.ExitCode
 	jt.Skipped = t.Skipped
 
 	// Set canceled flag on the job if a task was canceled through the context
 	if errors.Is(t.Error, context.Canceled) {
-		j.Canceled = true
+		jt.Canceled = true
+	} else {
```

Comment:
We shouldn't show tasks that were canceled as having an error, so we just set the `Canceled` flag in this case.